docker-compose.yml

```diff
@@ -17,10 +17,12 @@ services:
         condition: service_healthy
     build:
       context: ./packages/starksheet-cairo
+    volumes:
```

Comment:
the deployments folder need to be bridged to the host, the webapp uses the artifacts to compute the `contract.json`